vtex/billing-app
# Update/contract

#### What is the purpose of this pull request?
Save contracts populating the `name` field with the app's `title`

#### What problem is this solving?
In billing contracts, the name displayed is the contract's `name` field. For the sake of user's readability, it should no longer be used to accommodate the full id string. The field `appId` is the correct one to use in this case

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in your workspace
2. Purchase/Install an app. Once you accept the displayed price and conditions, the contract will be generated.
3. Check your `{{account}}.myvtexdev.com/admin/billing/contracts` to see if the contract will be generated with the right name
4. Update your apps that use `vtex.billing` or the `Billing` itself to comply with these changes. They may break your apps, e.g. `apps-graphql`, `extension-store`

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/3827456/39775138-6144ed66-52d3-11e8-923e-b1604389bcf2.png)

#### Types of changes
